src/Controls/tests/TestCases.HostApp/Issues/Issue28986.xaml

```diff
@@ -0,0 +1,120 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="Maui.Controls.Sample.Issues.Issue28986"
+             x:Name="TestPage"
+             Title="SafeAreaGuides Test">
+    
+    <Grid x:Name="MainGrid" BackgroundColor="Red">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="*" />
+            <RowDefinition Height="Auto" />
+        </Grid.RowDefinitions>
+        
+        <!-- Content area with colored background to show safe area effects -->
+        <Grid Grid.Row="0" x:Name="ContentGrid">
+            <Border BackgroundColor="LightBlue" Stroke="Blue" StrokeThickness="4">
+                <VerticalStackLayout Spacing="20" Padding="20">
+                    <Label Text="SafeAreaGuides Test Page" 
+                           FontSize="24" 
+                           FontAttributes="Bold"
+                           HorizontalOptions="Center"
+                           TextColor="DarkBlue" />
+                    
+                    <Label Text="The red background shows areas outside safe area. The blue border shows the content area."
+                           FontSize="14"
+                           HorizontalOptions="Center"
+                           TextColor="DarkBlue" />
+                    
+                    <Label x:Name="CurrentSettingsLabel"
+                           Text="Current: All edges ignore safe area"
+                           FontSize="16"
+                           FontAttributes="Bold"
+                           HorizontalOptions="Center"
+                           TextColor="DarkBlue"
+                           AutomationId="CurrentSettings" />
+                    
+                    <Frame BackgroundColor="Yellow" Padding="10" Margin="20">
+                        <Label Text="This yellow frame should show safe area behavior effects. When safe area is respected, content stays away from screen edges, notches, and toolbars."
```

Comment:
This seems inaccurate

Because this yellow frame is in the middle of the screen it's never going to go inside regions of the screen that are covered by the notch or the bar at the bottom. 

This control would need to either be at the top or the bottom to work.